src/test/java/redis/clients/jedis/tests/commands/PublishSubscribeCommandsTest.java

```diff
@@ -65,6 +65,29 @@ public void onUnsubscribe(String channel, int subscribedChannels) {
 
   @Test
   public void pubSubChannels() {
+    // Since we are utilizing sentinel for the tests, there is an additional '__sentinel__:hello'
+    // channel that has subscribers.
+    final List<String> expectedActiveChannels = Arrays.asList("testchan1", "testchan2",
+      "testchan3", "__sentinel__:hello");
```

Comment:
```suggestion
    final List<String> expectedActiveChannels = Arrays.asList("testchan1", "testchan2",
      "testchan3");
```

Reply:
Not sure if you wanted me to make a change here since you said you prefer it as it is?  If we remove the sentinel channel and still check the list via `containsAll`, the assert will fail.

Reply:
Ok, that is what we currently do, so I will leave everything as is.  Let me know if you had something different in mind.

Reply:
No. Look carefully, we're currently currently doing `assertTrue(expectedActiveChannels.containsAll(activeChannels));`. I'm suggesting a reverse check.

Reply:
Oh my goodness.  I am so sorry.  Yes, I can change that. Should we remove the comment about the sentinel channel altogether or change it to explain why we're doing the reverse check?

Reply:
I've updated it. Let me know what you think.